aws/aws-sdk-php-symfony/1.3/config/packages/aws.yaml

```diff
@@ -0,0 +1,10 @@
+# Basic configuration for AWS SDK
+# Please visit http://docs.aws.amazon.com/aws-sdk-php/v3/guide/guide/configuration.html
+# for more information
+
+aws:
+    version: latest
+    region: us-east-1
+    credentials:
+        key: not-a-real-key
+        secret: "@@not-a-real-secret"
```

Comment:
`key` and `secret` should probably be env vars instead.

Reply:
Really helpful. Thanks